test/e2e/bots_test.go

```diff
@@ -416,6 +416,28 @@ func runBotTest(t *testing.T,
 				assert.NoError(t, err)
 			})
 		})
+
+		k8sPrefixTests := []string{"kubectl", "kc", "k"}
+		for _, prefix := range k8sPrefixTests {
+			t.Run(fmt.Sprintf("Get Pods with k8s prefix %s", prefix), func(t *testing.T) {
+				command := fmt.Sprintf("%s get pods", prefix)
```

Comment:
You have 2 options to make it "green" :D

First, why it's failing? Because the k3d cluster that is created on CI, is "fresh" and there are no resources in the `default` namespace.

You can either:
1. Change the expectation to
    
    `kubectl get pods` on `sample`
    ```
    No resources found in default namespace.
    ```
    so the test run will be simplified to:
    ```go
    t.Run(fmt.Sprintf("Get Pods with k8s prefix %s", prefix), func(t *testing.T) {
	    command := fmt.Sprintf("%s get pods", prefix)
	    expectedBody := codeBlock("No resources found in default namespace.")
	    expectedMessage := fmt.Sprintf("%s\n%s", cmdHeader(command), expectedBody)
    
	    botDriver.PostMessageToBot(t, botDriver.Channel().Identifier(), command)
	    err = botDriver.WaitForLastMessageEqual(botDriver.BotUserID(), botDriver.Channel().ID(), expectedMessage)
	    assert.NoError(t, err)
    })
    ```

1. Run this command but in BotKube namespace, so just replace command with:
    ```go
    command := fmt.Sprintf("%s get pods --namespace %s", prefix, appCfg.Deployment.Namespace)
    ```